packages/plugins/Gfi/src/types.ts

```diff
@@ -41,6 +43,7 @@ export interface GfiGetters extends GfiState {
   gfiConfiguration: GfiConfiguration
   /** all layer keys to retrieve GFI information for */
   layerKeys: string[]
+  renderType: RenderType
```

Comment:
`windowLayerKeysActive`, `listMode`, `listText`, `showList` and `listFeatures` are missing on `GfiGetters`

Reply:
c330f31ad043f5220a0e57b55627fd1a22cf1882